README.md

```diff
@@ -2,68 +2,25 @@
 
-
-#### Automated (OSX)
-
-If you are developing on OSX, you can run the automated setup script. From
-the `vets-api` directory, run `./bin/setup-osx && source ~/.bash_profile && cd .`
-
-#### Alternative (OSX)
-
-1. Install Ruby 2.3.
-   - It is suggested that you use a Ruby version manager such as
-    [rbenv](https://github.com/rbenv/rbenv#installation) and
-    [install Ruby 2.3](https://github.com/rbenv/rbenv#installing-ruby-versions).
-   - *NOTE*: rbenv will also provide additional installation instructions in the
-    console output. Make sure to follow those too.
-1. Install Bundler to manage dependencies
-   - `gem install bundler`
-1. Install Postgres and enable on startup
-   - `brew install postgres`
-   - `brew services start postgres`
-1. Install Redis
-   - `brew install redis`
-   - Follow post-install instructions to enable Redis on startup. Otherwise,
-    launch it manually with `brew services start redis`.
-1. Install ImageMagick
-   - `brew install imagemagick`
-1. Install ClamAV
-  - `brew install clamav`
-  - Take note of the the post-install instructions `To finish installation & run clamav
-  you will need to edit the example conf files at ${conf_files_dir}` (_${conf_files_dir}_ 
-  will differ based on your homebrew location) then:
-    - `cd ${conf_files_dir}`
-    - `touch clamd.sock`
-    - `echo "LocalSocket ${conf_files_dir}" > clamd.conf` 
-    - `echo "DatabaseMirror database.clamav.net" > freshclam.conf`
-    - `freshclam -v`
-1. Install pdftk
-  - `brew install https://gist.githubusercontent.com/lihanli/03ec8f17a6a1ff52e3a149be4cf7f2ae/raw/d18eff01396bbc25a928f756ff21edcc3521fc5e/pdftk.rb`
-1. Install gem dependencies: `cd vets-api; bundle install`
-1. Install overcommit `overcommit --install --sign`
+1. Install [Docker for Mac](https://docs.docker.com/docker-for-mac/install/). This will configure both `docker` and `docker-compose`.
 1. Setup localhost certificates / keys:
-   - Create a hidden folder in your home directory:  `mkdir ~/.certs`
-   - Copy the [certificate][certificate] to `~/.certs/vetsgov-localhost.crt`
-   - Copy the [key][key] to `~/.certs/vetsgov-localhost.key`
+   - Create a folder in your vets-api directory:  `mkdir config/certs`
+   - Copy the [certificate][certificate] to `config/certs/vetsgov-localhost.crt`
+   - Copy the [key][key] to `config/certs/vetsgov-localhost.key`
    - *NOTE*: If you don't have access to these keys, running the following
-     commands will provide basic functionality, such as for running unit tests:
-   - `touch ~/.certs/vetsgov-localhost.crt`
-   - `touch ~/.certs/vetsgov-localhost.key`
-1. Create dev database: `bundle exec rake db:setup`
+     commands will provide basic functionality:
+   - `touch config/certs/vetsgov-localhost.crt`
+   - `touch config/certs/vetsgov-localhost.key`
+1. Run the vets-api dependencies and application
+    - `make up`
 
+The API will then be available on port 3000 of the docker host.
```

Comment:
Maybe add a bit of context around Docker for those who aren't familiar. E.g if I change code do I need to run `make up` again, or are changes automatically reflected in the container? It's not obvious to everybody.